importer/balanced/builder.go

```diff
@@ -262,3 +265,108 @@ func fillNodeRec(db *h.DagBuilderHelper, node *h.FSNodeOverDag, depth int) (fill
 
 	return filledNode, nodeFileSize, nil
 }
+
+func LayoutEnc(db *h.DagBuilderHelper, jwe []byte) (ipld.Node, error) {
+	fmt.Println("inside LayoutEnc rrrrrrrrrrrrrrrrrrr", jwe)
+	if db.Done() {
+		fmt.Println("inside LayoutEnc rrrrrrrrrrrrrrrrrrrdddd", jwe)
```

Comment:
Consider using https://github.com/ipfs/go-log

Please have a search for `fmt.Println` across the PR and replace as needed.

Reply:
done